pkg/api/v1/pod/util.go

```diff
@@ -406,3 +406,12 @@ func UpdatePodCondition(status *v1.PodStatus, condition *v1.PodCondition) bool {
 	// Return true if one of the fields have changed.
 	return !isEqual
 }
+
+// IsRestartableInitContainer returns true if the initContainer has
+// ContainerRestartPolicyAlways.
```

Comment:
make sure to explicitly say that this method is not checking that the container passed to it is indeed an init container.

Logically, we only allow to set `RestartPolicy` on the restartable init container. However it may change in future and we will not have a good way to catch a problem if somebody start using this method to check that the container IS a init container.

Reply:
Done.